client/dfclient/deadfish.ini.example

```diff
@@ -0,0 +1,6 @@
+[default]
+client_mode=matchmaker
+matchmaker_address=localhost
+matchmaker_port=8000
+direct_address=localhost
+direct_port=63987
```

Comment:
The viable options are not documented anywhere. Maybe some comments here could be a good idea, as it is an example file? But anywhere would be ok, even in the [README.md](https://github.com/capturcus/deadfish/blob/master/README.md).

Reply:
Added coments in the ini example file